src/locales/ca/en/index.json

```diff
@@ -0,0 +1,34 @@
+{
+  "alreadyHaveAnAccount": "Already have an account? <a href='/'>Sign in here!</a>",
+  "dontHaveAnAccount": "Don’t have an account? <a href='/register'>Sign up here!</a>",
+  "closeModal": "Close Modal",
+  "chooseDifferentUsername": "Please choose a different username.",
+  "ensurePasswordIsMoreThan": "Ensure password contains more than {minPasswordLength} characters.",
+  "ensurePasswordHasOne": "Ensure password contains 1 of each: capital letter, number and special character.",
+  "givenName": "First Name",
+  "loading": "Loading ...",
+  "loginButton": "Login In",
+  "loginHeader": "Login In",
+  "loginSuccess": "Login Success!",
```

Comment:
We have a message for login success, but nothing for login failure. The app does not seem to render any form of failure page, instead rendering the login page again. Is there a better way we can communicate to the user what the problem might be on login?

Reply:
Step submission failures are not yet handled within the app. That's what I'm working on this sprint, so I'll add any localizations to the app through that work.